frame/executive/src/lib.rs

```diff
@@ -933,7 +937,10 @@ mod tests {
 	fn block_weight_limit_enforced() {
 		let mut t = new_test_ext(10000);
 		// given: TestXt uses the encoded len as fixed Len:
-		let xt = TestXt::new(Call::Balances(BalancesCall::transfer(33, 0)), sign_extra(1, 0, 0));
+		let xt = TestXt::new(
+			Call::Balances(BalancesCall::transfer { dest: 33, value: 0 }),
```

Comment:
Huh, not using `call_transfer` here?

Reply:
I am just using what was already there, modified to the struct variant syntax. Not sure whether what was already there is correct or not...